IdentityCore/tests/MSIDAADRequestErrorHandlerTests.m

```diff
@@ -107,7 +107,7 @@ - (void)testHandleError_whenItIsServerError_shouldRetryRequestAndDecreseRetryCou
 
     [self keyValueObservingExpectationForObject:httpRequest keyPath:@"sendWithBlockCounter" expectedValue:@1];
 
-    [self.errorHandler handleError:[NSError new]
+    [self.errorHandler handleError:[NSError errorWithDomain:NSURLErrorDomain code:NSURLErrorNetworkConnectionLost userInfo:nil]
```

Comment:
this test case is for testing 5xx error code, so should we change to some other error such that it can go through to the 5xx error handle logic? ;)